Tests/UnitTests/Mocks/Sessions+Services/Storage/MockDefaultsStore.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+@testable import OneLogin
+
+class MockDefaultsStore: DefaultsStorable {
+    var savedData = [String: Any]()
+    var returningAuthenticatedUser: Bool?
+    var returnExpDate: Date?
+    
+    func set(_ value: Any?, forKey defaultName: String) {
+        savedData[defaultName] = value
+    }
+    
+    func value(forKey key: String) -> Any? {
```

Comment:
Kind of a nitpick but this has a bit of a code smell to me.  For example, if in `test_getAccessToken_fails()` you assert
`XCTAssertNil(mockDefaultStore.value(forKey: .accessTokenExpiry))`
instead of 
`XCTAssertNil(mockDefaultStore.savedData[.accessTokenExpiry])`
the test fails.  Which does not seem right.

And as our list of mock items grows, this might become a bit unmanageable.   Any reason not to just go with 

```
func value(forKey key: String) -> Any? {
  savedData[key]
}
```

Reply:
Good shout, wood from the trees situation there I think 😅, made that change now 👍